src/AutoMerge/guidelines.jl

```diff
@@ -619,17 +619,22 @@ const guideline_src_names_OK = Guideline(;
 )
 
 function meets_code_can_be_downloaded(registry_head, pkg, version, pr; pkg_code_path)
-    uuid, package_repo, subdir, tree_hash_from_toml = parse_registry_pkg_info(
+    uuid, package_repo, subdir, tree_hash_from_toml, commit_hash_from_toml, tag_name_from_toml = parse_registry_pkg_info(
```

Comment:
Seems like now is as good a time as any to make the following change:

```suggestion
    (; uuid, package_repo, subdir, tree_hash_from_toml, commit_hash_from_toml, tag_name_from_toml) = parse_registry_pkg_info(
```

Reply:
i agree, but they aren't the names....

Reply:
Ah, that's annoying.

Reply:
Also, this was only added in Julia 1.7.